README.md

```diff
@@ -219,6 +215,18 @@ $ ls bin/local
 ecs-cli
 ```
 
+### Adding a new dependency
+
+* Make sure you the latest godep (go get -u github.com/tools/godep)
+* `go get` the new dependency.
+* Edit your application’s source code to import the new dependency.
+* Run `godep save ./...` to update Godeps/Godeps.json and copy the dependencies source to vendor/.
+
+### Updating an existing dependency
```

Comment:
Also, it seems to me at this point that we have to use godep restore when you first pull down the package. I am not able to update a dependency and get godep to work using these instructions. They do seem to work if `godep restore` is run when you close the CLI for the first time.

Unfortunately, using godep restore means that `go get` will not work with dependencies. So for now when we want to update a dependency we will have to go to the dependency in the `GOPATH` and manually use git pull an update to that package.